packages/tools/perftest/src/cases/PODTimer.ts

```diff
@@ -90,3 +90,21 @@ export class PODVerifyCase extends TimerCase {
     await PODPCDPackage.verify(this.pcds[iteration]);
   }
 }
+
+export class PODGetMerkleTreeCase extends TimerCase {
+  constructor() {
+    super("pod-pcd.getMerkleTree");
+  }
+
+  async init(): Promise<void> {
+    await PODPCDPackage.init?.({});
+  }
+
+  async setup(_: number): Promise<void> {}
+
+  async op(_: number): Promise<void> {
+    const podContent = PODContent.fromEntries(sampleEntries);
+    // Ensure that we use the contentID somehow
+    console.log(podContent.contentID);
```

Comment:
Console logging can sometimes cause unpredictable time delays.  If you want to ensure the contentID is used for something which can't be optimized out (is optimization even a thing in TS?) maybe something like this?

```ts
  if (podContent.contentID === undefined) {
    throw new Error(...);
  }
```

Reply:
Done!